src/PIL/PpmImagePlugin.py

```diff
@@ -135,7 +157,7 @@ def _save(im, fp, filename):
     elif im.mode == "RGBA":
         rawmode, head = "RGB", b"P6"
     else:
-        raise OSError(f"cannot write mode {im.mode} as PPM")
+        raise OSError(f"Cannot write mode {im.mode} as PPM")
```

Comment:
```suggestion
        raise OSError(f"cannot write mode {im.mode} as PPM")
```
Considering that "cannot write mode" is used in 9 other plugins, it would be nice not to have this one be unique.